src/ansys/fluent/core/session.py

```diff
@@ -111,7 +111,9 @@ def build_from_fluent_connection(self, fluent_connection: FluentConnection):
         self.error_state = self.fluent_connection.error_state
         self.scheme_eval = self.fluent_connection.scheme_eval
         self.rp_vars = RPVars(self.scheme_eval.string_eval)
-        self._uploader = None
+        self._file_transfer_service = FileTransferService(
```

Comment:
It would be really nice if this service were actually passed in to the constructor. We can leave that as a separate piece of work because we can look at the other dependencies too.

Reply:
Done. Thank you.